components/dashboard/src/App.tsx

```diff
@@ -163,6 +32,7 @@ function App() {
         }
     }, []);
 
+    // TODO: Add some context to what this logic is for
```

Comment:
This is so that if someone edits the URL adding a hash we reload the page. I guess the idea was to not forcing users to explicitly reload. I don't think this is used or important.